### PR TITLE
Make random example more interesting.

### PR DESCRIPTION
**Pull request by juliusv**
Have different RPC rates for the three services and vary them according
to a diurnal period.

### CODE REVIEW

**Comment by juliusv** [comment]
@beorn7

File: examples/random/main.go [hunk 1]
```diff
@@ -27,9 +28,10 @@ import (
 
 var (
 	addr          = flag.String("listen-address", ":8080", "The address to listen on for HTTP requests.")
-	uniformDomain = flag.Float64("random.uniform.domain", 200, "The domain for the uniform distribution.")
-	normDomain    = flag.Float64("random.exponential.domain", 200, "The domain for the normal distribution.")
-	normMean      = flag.Float64("random.exponential.mean", 10, "The mean for the normal distribution.")
+	uniformDomain = flag.Float64("uniform.domain", 200, "The domain for the uniform distribution.")
+	normDomain    = flag.Float64("exponential.domain", 200, "The domain for the normal distribution.")
+	normMean      = flag.Float64("exponential.mean", 10, "The mean for the normal distribution.")
+	diurnalPeriod = flag.Duration("diurnal-period", 10*time.Minute, "The duration of a diurnal oscillation period.")
```
Comment (beorn7) [inline examples/random/main.go]: How does _diurnal_ make sense here?

Comment (juliusv) [inline examples/random/main.go]: It's how long you want your day to be :) But I can give it a different name if you want...

Comment (beorn7) [inline examples/random/main.go]: Like _oscillation-period_ or even just _period_?

Comment (juliusv) [inline examples/random/main.go]: Changed to oscillation-period.

**Comment by beorn7** [comment]
:+1:

**Pull request merged**